code/datums/quirks/neutral.dm

```diff
@@ -299,6 +299,18 @@
 		)
 	)
 
+/datum/quirk/item_quirk/finnish
+	name = "Finland Enthusiast"
+	desc = "You have a powerful relic of old Finland in your posession."
+	value = 0
+	mob_trait = TRAIT_FINNABOO
+	gain_text = "<span class='notice'>You know enjoy everything about Finland.</span>"
```

Comment:
"know enjoy"
grammar [CENSORED FOR GIT] mode activated

Reply:
bazinga